src/Diagnostics.RuntimeHost/Services/SourceWatcher/Watchers/StorageWatcher.cs

```diff
@@ -50,7 +53,7 @@ private bool LoadOnlyPublicDetectors
         }
           
 
-        public StorageWatcher(IHostingEnvironment env, IConfiguration config, IStorageService service, IInvokerCacheService invokerCache, IGistCacheService gistCache)
+        public StorageWatcher(IHostingEnvironment env, IConfiguration config, IStorageService service, IInvokerCacheService invokerCache, IGistCacheService gistCache, IKustoMappingsCacheService kustoCache)
```

Comment:
Just a small nitpik to rename `kustoCache` to `kustoMappingsCache`. The naming could be misleading.